main.go

```diff
@@ -199,8 +201,14 @@ func initSearch(out io.Writer, o *searchRepoOptions) (*search.Index, error) {
 // It will return a Pointer to the Chart Result (the Pointer points to the
 // Result of the Index).
 // If no results are found, nil will be returned instead of type *Result.
-func searchChart(r []*search.Result, name string, chartVersion string, devel bool) (*search.Result, error) {
-	found := false // found describres if Charts where found but no one is newer than the actual one
+// And the bool describes if it may be some Repositories contain a deprecated chart.
+func searchChart(r []*search.Result, name string, chartVersion string, devel bool) (*search.Result, bool, error) {
+	//  since we have now to check also if a repository contains an
+	// deprecated chart we need an "point" where to look if we have found
+	// a newer chart version
+	foundNewer := false
+	found := false                 // found describres if Charts where found but no one is newer than the actual one
```

Comment:
Typo in comments : "describres" should be "describes"

Reply:
FIxed with Commit 7db5c50ad510425a3586ff62afdde7c98d50c514